lib/bundler/ui/shell.rb

```diff
@@ -80,6 +81,9 @@ def silence
 
       # valimism
       def tell_me(msg, color = nil, newline = nil)
+        return if @told_messages.include? msg
```

Comment:
I can make that happen if you guys want to merge this way. 

Those are the levels currently:
```
LEVELS = %w(silent error warn confirm info debug)
```
Confirming, should I filter only by warn level?